tools/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# ~~~
+# Create a header-only library from an SVD register definition file.
+#
+# add_groov_reg_lib(
+#     groov_reg_test
+#     INPUT_FILE "${CMAKE_SOURCE_DIR}/regs/rp2040.svd"
+#     OUTPUT_DIR "${CMAKE_BINARY_DIR}/regs"
+#     PERIPHERALS adc rtc
+# )
+# ~~~
+
+function(add_groov_reg_lib target_name)
+    message(ERROR "Python3 not found, `add_groov_reg_lib` is not available.")
+endfunction()
+
+if(NOT Python3_FOUND)
+    return()
+endif()
+
+function(add_groov_reg_lib target_name)
+    set(options "")
+    set(oneValueArgs INPUT_FILE OUTPUT_DIR)
+    set(multiValueArgs PERIPHERALS)
+    cmake_parse_arguments(PARSE_ARGV 1 ARG "${options}" "${oneValueArgs}"
+                          "${multiValueArgs}")
+
+    string(JOIN " " peripherals_string "${ARG_PERIPHERALS}")
+    set(python_script "${CMAKE_CURRENT_FUNCTION_LIST_DIR}/regs2groov.py")
+    set(register_headers)
+
+    set(register_headers)
```

Comment:
**suggestion**: Remove lines 29 & 31, they aren't needed.

Reply:
Huh! I don't know how that got duplicated...I'll remove them.